apps/web/src/pages/AdminDashboardPage/AdminDashboardPage.tsx

```diff
@@ -68,70 +85,114 @@ const DashboardPage = ({ currentUser }: DashboardPageProps) => {
           },
         )}
       </PageHeader>
-      <p>
+      <Heading
+        size="h4"
+        data-h2-font-weight="base(bold)"
+        data-h2-margin="base(0, 0, x1, 0)"
+      >
         {intl.formatMessage({
-          defaultMessage:
-            "On this page you can find a list of active pools along with a few details about their status.",
-          id: "7B+1RO",
+          defaultMessage: "What are you working on today?",
+          id: "wHnGAK",
           description:
-            "Description of the content found on the admin portal dashboard page.",
+            "Heading for the sections of links available to the current user",
         })}
-      </p>
+      </Heading>
       <div
         data-h2-display="base(flex)"
-        data-h2-flex-wrap="base(wrap)"
-        data-h2-margin="base(x2, 0, 0, 0)"
+        data-h2-flex-direction="base(column)"
+        data-h2-gap="base(x1, 0)"
       >
-        <span data-h2-margin="base(0, x.5, x.5, 0)">
-          <IconLink
-            mode="solid"
-            color="secondary"
-            type="button"
-            href={adminRoutes.poolTable()}
-            icon={Squares2X2Icon}
-          >
-            {intl.formatMessage({
-              defaultMessage: "Manage pools",
-              id: "HgQThJ",
-              description:
-                "Text label for link to pools page on admin dashboard",
+        {hasRole("pool_operator", roleAssignments || []) && (
+          <LinkWell
+            title={intl.formatMessage({
+              defaultMessage: "Managing a recruitment process",
+              id: "293bsq",
+              description: "Heading for pool operator dashboard links",
             })}
-          </IconLink>
-        </span>
-        <span data-h2-margin="base(0, x.5, x.5, 0)">
-          <IconLink
-            mode="solid"
-            color="secondary"
-            type="button"
-            href={adminRoutes.userTable()}
-            icon={UserIcon}
-          >
-            {intl.formatMessage({
-              defaultMessage: "Manage users",
-              id: "U5HvOH",
-              description:
-                "Text label for link to users page on admin dashboard",
+            links={[
+              {
+                label: intl.formatMessage(adminMenuMessages.pools),
+                href: adminRoutes.allPools(),
+                icon: Squares2X2Icon,
+              },
+              {
+                label: intl.formatMessage({
+                  defaultMessage: "My Teams",
+                  id: "N3uD4m",
+                  description: "Link text for current users teams page",
+                }),
+                href: adminRoutes.teamTable(),
+                icon: BuildingOffice2Icon,
+              },
+            ]}
+          />
+        )}
+        {hasRole("request_responder", roleAssignments || []) && (
```

Comment:
I'm not a big fan of having to add `|| []` if the roleAssignments array is null/undefined. Can we handle that in the `hasRole()` function instead?

Reply:
Great suggestion! Applied in [allow hasRole to handle undefined roles](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6096/commits/cd0ce8fbc92b027f4775e8cb5ec601f0de27785a)